Content.Shared/Clumsy/ClumsyComponent.cs

```diff
@@ -0,0 +1,55 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Clumsy;
+
+/// <summary>
+/// A simple clumsy tag-component.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class ClumsyComponent : Component
+{
+
+    // Standard options. Try to fit these in if you can!
+
+    /// <summary>
+    ///     Sound to play when clumsy interactions fail.
+    /// </summary>
+    [DataField]
+    public SoundSpecifier ClumsySound = new SoundPathSpecifier("/Audio/Items/bikehorn.ogg");
+
+    /// <summary>
+    ///     Default chance to fail a clumsy interaction.
+    ///     If a system needs to use something else, add a new variable in the component, do not modify this percentage.
+    /// </summary>
+    [DataField]
+    public float ClumsyDefaultCheck = 0.5f;
+
+    /// <summary>
+    ///     Default stun time.
+    ///     If a system needs to use something else, add a new variable in the component, do not modify this number.
+    /// </summary>
+    [DataField]
+    public TimeSpan ClumsyDefaultStunTime = TimeSpan.FromSeconds(2.5);
+
+    // Specific options
+
+    /// <summary>
+    ///     Sound to play after hitting your head on a table. Ouch!
+    /// </summary>
+    [DataField]
+    public SoundCollectionSpecifier TableBonkSound = new SoundCollectionSpecifier("TrayHit");
```

Comment:
Should this maybe stay part of BonkableComponent? Climbable thing other than tables might make another sound.

Reply:
I really wish I could have figured out a way to remove the BonkableComponent, its super weird and only exists because of the different damages depending on which table you hit your head on 😭 

The reason its here and not the boinkable component is because technically you can clime onto something that *isn't* a table and doesn't inherit the boinkable component, and I still would want them to get stuned and make the noise (Just not take damage).

The `HitHeadOnTableClumsy` is a poor name because it actually doesn't have to be a table (However, currently I think tables are the only thing you can climb on).

Reply:
Good point, should be ok then